unity-client/Assets/Rendering/Culling/CullingController.cs

```diff
@@ -330,17 +331,17 @@ internal void ResetObjects()
             var renderers = objectsTracker.GetRenderers();
             var animations = objectsTracker.GetAnimations();
 
-            for (var i = 0; i < skinnedRenderers.Length; i++)
+            for (var i = 0; i < skinnedRenderers?.Length; i++)
```

Comment:
Only for curiossity: why did you need to add the `?` operator? Were we receiving sometimes the `skinnedRenderers` as null and it maked the function flow fail?

Reply:
I don't think those array can be null in runtime, but there were throwing an annoying error when stopping unity editor :)